config-templates/netapp/netapp-trident/changelog.json

```diff
@@ -3,11 +3,22 @@
         "22.04": {
             "status": "supported",
             "latest-revision": {
+                "revision": "3",
+                "release-date": "13 January 2023",
+                "cves": "",
+                "golang": "1.18.8",
+                "ubi": "8.7-923.1669829893",
+                "new-features": [],
+                "fixes": [
+                    "Use the policy/v1 API version for PodSecurityPolicy"
+                ]
+            },
+            "previous-revision": {
                 "revision": "2",
                 "release-date": "18 July 2022",
                 "cves": "",
-                "golang": "",
-                "ubi": "",
+                "golang": "1.18.8",
+                "ubi": "8.7-923.1669829893",
```

Comment:
same here

Reply:
Addressed in https://github.com/IBM/ibm-satellite-storage/pull/266/commits/b8f8c2e3ff4caac6b4a2c99e1b2d31fc64896345.